docs/renarde/JavaSupport.md

```diff
@@ -0,0 +1,14 @@
+# Java Support
+
+### REST Endpoints
+
+For Renarde `Controller` classes, Renarde uses a different strategy than JAX-RS to determine which methods count as REST endpoints, and what the path of each endpoint is.
```

Comment:
For [Renarde Controller](https://quarkiverse.github.io/quarkiverse-docs/quarkus-renarde/dev/index.html#_controllers) classes.